test/scraping/cycling/procyclingstats/raceStageResults.test.js

```
@@ -0,0 +1,32 @@
+    year: 2025,
+    stage: 1,
+    input:
+      "test/scraping/cycling/procyclingstats/html/raceStageResults-tour-down-under-1.html",
+    output:
+      "test/scraping/cycling/procyclingstats/fixtures/raceStageResults-2025-tour-down-under-1.json",
+  },
+])(`$race - $year - Stage $stage`, (data) => {
+  let html, expectedResults;
+
+  beforeAll(async () => {
+    const input = Bun.file(data.input);
```

ESLint

[error] 'Bun' is not defined.

(no-undef)